.travis.yml

```diff
@@ -5,7 +5,7 @@ before_script:
   - sh -e /etc/init.d/xvfb start
   - bundle install
 env:
-  - DISPLAY=':99.0' RAILS_VERSION='3.2.0'
-  - DISPLAY=':99.0'  RAILS_VERSION='4.1.0'
-  - DISPLAY=':99.0'  RAILS_VERSION='4.2.0'
+  - DISPLAY=':99.0' RAILS_VERSION='4.1.0'
+  - DISPLAY=':99.0' RAILS_VERSION='4.2.0'
+  - DISPLAY=':99.0' RAILS_VERSION='5.0.0'
```

Comment:
I think that's what we expect it to do, but I don't think that's what it does

![image](https://cloud.githubusercontent.com/assets/351915/17557028/9a09e4d4-5ee3-11e6-9974-0583955c1778.png)

![image](https://cloud.githubusercontent.com/assets/351915/17557007/8a2fe734-5ee3-11e6-8fb9-09f347df6fa5.png)